light_curves/requirements_light_curve_generator.txt

```diff
@@ -14,6 +14,9 @@ astroquery>=0.4.8.dev0
 acstools
 lightkurve
 alerce
+lsdb
+s3fs
```

Comment:
We don't directly use s3fs so don't need to add it to this file.

Also do we have to manually install it as it's an undeclared dependency of lsdb? (would be nice to be able to pick it up automatically though an extras, although it's a theoretical ask for the lsdb developers as here we already install it as part of `pandas[aws]`

Reply:
I removed s3fs from the requirements file.  I don't quite understand what the ask is for lsdb, so please open an issue over there if it is desirable.

Reply:
xref: https://github.com/astronomy-commons/lsdb/issues/436